static/styles/zulip.css

```diff
@@ -2389,21 +2392,20 @@ button.topic_edit_cancel {
 .message_embed .message_embed_title {
     padding-top: 0px;
     /* to remove the spacing that the font has from the top of the container. */
-    margin-top: -5px;
+    margin-top: -1px;
 
     font-size: 1em;
     font-weight: 600;
     line-height: normal;
 }
 
-.message_embed .message_embed_title a:hover {
-    text-decoration: none;
-    border-bottom: 1px solid #005580;
-}
-
 .message_embed .message_embed_description {
+    position: relative;
     margin-top: -5px;
     max-width: 500px;
+
+    /* to put it below the container gradient. */
+    z-index: -1;
```

Comment:
No, it's not necessary anymore. It is inert at this point since it is the only layer in that scope.